File: Modul-4/Soal-5/PRAK405-2410817110016-MuhammadAzriRahman.py
baris, kelipatan = map(int, input("Masukkan jumlah baris dan kelipatan: ").split())

total = 0
    

for i in range(1, baris + 1):
    hasil_penjumlahan = 0
        
    for j in range(i, 0, -1):
        hasil_penjumlahan += j * kelipatan
        print(f"({j} * {kelipatan})", end="")
        if j > 1:
            print(" + ", end="")
        
    print(f" = {hasil_penjumlahan}")
        
    total += hasil_penjumlahan
    

print(total)

File: Modul-4/Soal-5/PRAK405-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>

void main() {
    int baris, kelipatan;

    printf("Masukan nilai baris dan kelipatan: ");
    scanf("%d %d", &baris, &kelipatan);

    int total = 0;

    for (int i = 1;i <= baris;i++) {
        int hasil_penjumlahan = 0;
        
        for (int j = i;j >= 1;j--) {
            hasil_penjumlahan += j*kelipatan;
            printf("(%d * %d)",j,kelipatan);
            if (j > 1) {
                printf(" + ");
            }
        }
        printf(" = %d\n", hasil_penjumlahan);
        total += hasil_penjumlahan;
    }
    printf("%d", total);
}